test/index.js

```diff
@@ -310,6 +321,15 @@ describe('client', async () => {
 
       client.done()
     })
+
+    it('throws error when provided', { plan: 1 }, async () => {
+      const client = new NoGres.Client()
+      await client.connect()
+      const { sql, params, returns, throws } = client.expect('select * from orders where id = $1', [1, 2, 3], new Error('example error'))
```

Comment:
The `returns` and `throws` are not needed here anymore, they throw a linting error `<var> is assigned a value but never used`. They can be removed since error throwing is already checked by the test.
```suggestion
      const { sql, params } = client.expect('select * from orders where id = $1', [1, 2, 3], new Error('example error'))
```

Reply:
haha what timing! I will address that in a followup PR, thanks!

Reply:
Yes,... great timing...can you believe it?  😂